TemplateFiles/DeploymentFiles/runtime-configuration-initial.js

```diff
@@ -0,0 +1,9 @@
+window.__UI_CONFIGURATION_INITIAL__ = {
+    "tenantId": "e60f12c0-e1dc-4be1-8d86-e979a5527830",
```

Comment:
why this value coming up, are these actual values.

Reply:
Removed the file. It got pushed by mistake.